add tests to logger package

43081j opened the pull request:
felt like adding some tests to this package, let me know if you think they're worth merging in.

couple of oddities i noticed along the way you might want to think about in future:

* most of your tests are in JS but can be in typescript as you already pass them through 'ts-node'
* your types (e.g. `@types/mocha`) are often or always unused because of this
* weirdly you get the older (typescript <3.1) sinon types, im not too sure this is as you're on a recent enough typescript, maybe its some mistake by ts-node

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/slackapi/node-slack-sdk?pullRequest=804) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/slackapi/node-slack-sdk?pullRequest=804) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/slackapi/node-slack-sdk?pullRequest=804) it.</sub>

aoberoi commented on packages/logger/src/index.spec.ts:
```diff
@@ -0,0 +1,134 @@
+import { assert } from 'chai'; // tslint:disable-line:no-implicit-dependencies
```
this PR also pushes for tests to be in TypeScript, and [we want to move that decision](https://github.com/slackapi/node-slack-sdk/pull/808#issuecomment-530593159) outside unrelated PRs.

aoberoi commented:
this is really awesome @43081j, thanks for taking the time to implement the tests!

43081j commented:
@aoberoi since i made this PR, a couple of tests got added.

i've thrown away the typescript change, rebased onto master and tried to merge the existing tests with mine. there was a confusing one in there which i deleted (see the diff) as i _think_ whatever it was trying to assert is covered by the other tests

seratch commented:
The change in this PR has been merged by #1269 - Thanks for your contribution!
